git-extra/env.sh

```diff
@@ -20,14 +20,12 @@ test -n "$SSH_ASKPASS" || {
 			export SSH_ASKPASS=/mingw32/libexec/git-core/git-gui--askpass
 		fi
 		;;
-	ARM64)
+	CLANGARM64)
 		export DISPLAY=needs-to-be-defined
-		if [ -f /arm64/bin/git-askpass.exe ]; then
-			export SSH_ASKPASS=/arm64/bin/git-askpass.exe
-		elif [ -f /mingw32/bin/git-askpass.exe ]; then
-			export SSH_ASKPASS=/mingw32/bin/git-askpass.exe
+		if [ -f /clangarm64/bin/git-askpass.exe ]; then
+			export SSH_ASKPASS=/clangarm64/bin/git-askpass.exe
```

Comment:
This is currently built with `CC=${MINGW_CHOST}-gcc`. I guess that's internally handled by `clang` on ARM64 through some compatibility layer. Otherwise we might need to tweak the `PKGBUILD` to build those executables natively.

Reply:
There's a compatibility layer indeed: [`gcc-compat`](https://packages.msys2.org/package/mingw-w64-clang-aarch64-gcc-compat?repo=clangarm64). This ensures that things are handled by `clang` internally. `git-extra` [builds just fine](https://github.com/git-for-windows/build-extra/pull/455) using Clang 👍🏼 

Moving forward, we should update that to `CC=${MINGW_CHOST}-cc` instead, so that either GCC or Clang is used, depending on what `MSYSTEM` we're in. The MSYS2 team has done that for basically all `MINGW-packages` ~1 year ago: https://github.com/msys2/MINGW-packages/commit/7c67d9c46554d10276b2017bcfd0290dfe24a1bf